### PR TITLE
Adds Fluff Item "Tyhailer"

### PR DESCRIPTION
**Pull request by Ty-Omaha**
A sec gasmask that allows me to choose what shitcurity phrase it will say with the "adjust mask" button

:cl: :
:rscadd: Adds Tyhailer Fluff Item
/:cl:

### CODE REVIEW

**Comment by FlattestGuitar** [comment]
This is way too OP. 

👎👎👎👎❌💯

**Comment by Ty-Omaha** [comment]
You're way too OP. :-1: :-1: :-1: :heavy_multiplication_x: :100: 

:(

File: code/modules/customitems/item_defines.dm [hunk 1]
```diff
@@ -426,6 +426,71 @@
 
 //////////// Masks ////////////
 
+obj/item/clothing/mask/gas/fluff/tyhailer // T0EPIC4U: Ty Omaha
+	name = "tacticool security gas mask"
+	desc = "A tacticool Security gas mask with modified 'Compli-o-nator 3000' device, let's Ty be the snowflake he is and choose the phrases it announces. It looks like it has some..residue..you probably don't want to touch it."
```
Comment (FlattestGuitar) [inline code/modules/customitems/item_defines.dm]: a) Probably don't want to mention Ty specifically, as it's not really related to him backstory-wise. 
b) lets*
c) 😏

Comment (Ty-Omaha) [inline code/modules/customitems/item_defines.dm]: Is changeds just for yousies yaya!

File: code/modules/customitems/item_defines.dm [hunk 2]
```diff
@@ -426,6 +426,71 @@
 
 //////////// Masks ////////////
 
+obj/item/clothing/mask/gas/fluff/tyhailer // T0EPIC4U: Ty Omaha
+	name = "tacticool security gas mask"
+	desc = "A tacticool Security gas mask with modified 'Compli-o-nator 3000' device, it's labeled: Ty Omaha. It looks like it has some..residue..you probably don't want to touch it."
```
Comment (FlattestGuitar) [inline code/modules/customitems/item_defines.dm]: I really don't want to do this, but you're forcing my hand here. 

`It's labeled: "Ty Omaha". `

Comment (Ty-Omaha) [inline code/modules/customitems/item_defines.dm]: I would need 2 sets of quotation marks for that and I just tried doing ""Ty Omaha"" and I got errors doing that..

So yous be a good meatbag and be quiets! 

:(

Comment (Crazylemon64) [inline code/modules/customitems/item_defines.dm]: `It's labeled: \"Ty Omaha\".`
Backslashes prevent quotes from being read as REAL quotes.

Comment (Ty-Omaha) [inline code/modules/customitems/item_defines.dm]: shrieks!

**Comment by Iamgoofball** [comment]
why dont you just put this functionality into the default hailer

File: code/modules/customitems/item_defines.dm [hunk 3]
```diff
@@ -426,6 +426,71 @@
...
+				phrase_sound = "stfu"
+			if(6)
+				phrase_text = "I am, the LAW!"
+				phrase_sound = "dredd"
+			if(7)
+				phrase_text = "Dead or alive you're coming with me."
+				phrase_sound = "robocop"
+
+		usr.visible_message("[usr]'s tacticool Compli-o-Nator: <font color='red' size='4'><b>[phrase_text]</b></font>")
+		playsound(src.loc, "sound/voice/complionator/[phrase_sound].ogg", 100, 0, 4)
```
Comment (tigercat2000) [inline code/modules/customitems/item_defines.dm]: src. is unnecessary

File: code/modules/customitems/item_defines.dm [hunk 4]
```diff
@@ -426,6 +426,71 @@
...
+	var/phrase = 1
+	actions_types = list(/datum/action/item_action/halt, /datum/action/item_action/adjust)
+
+/obj/item/clothing/mask/gas/fluff/tyhailer/ui_action_click(mob/user, actiontype)
+	if(actiontype == /datum/action/item_action/halt)
+		tyhalt()
+	else
+		switch(phrase)
+			if(1)
+				to_chat(user, "\blue You set the special snowflake restrictor to: Stop or Bash")
```
Comment (tigercat2000) [inline code/modules/customitems/item_defines.dm]: On all of these, please don't use color macros, use span classes.
`to_chat(user, "<span class='notice'>You set the special snowflake restrictor to: Stop or Bash</span>")`

**Comment by LittleBigKid2000** [comment]
As much as I dislike goofball, I somewhat agree with him. It would probably be better to at least make this a subtype of the sechailer.

**Comment by Fox-McCloud** [comment]
Merge commits---not acceptable.

**Pull request closed without merging**